homeassistant/components/mqtt/strings.json

```diff
@@ -7,6 +7,14 @@
     "deprecation_mqtt_legacy_vacuum_discovery": {
       "title": "MQTT vacuum entities with legacy schema added through MQTT discovery",
       "description": "MQTT vacuum entities that use the legacy schema are deprecated, please adjust your devices to use the correct schema and restart Home Assistant to fix this issue."
+    },
+    "entity_name_is_device_name": {
+      "title": "MQTT entities with a name is equal to device name",
+      "description": "Some MQTT entities have an entity name that is equal to the device name. This is not expected. To avoid a duplicate names the entity name is set to `null` as a work-a-round. Please update your configuration, or inform the maintainer of the integration that supplies the entity, and restart Home Assistant to fix this issue.\n\nList of effected entities:\n\n{config}"
```

Comment:
```suggestion
      "description": "Some MQTT entities have an entity name that is equal to the device name. This is not expected. To avoid a duplicate name the entity name is set to `null` as a work-a-round. Please update your configuration, or inform the maintainer of the integration that supplies the entity, and restart Home Assistant to fix this issue.\n\nList of effected entities:\n\n{config}"
```